kirin/queue_workers/piv/piv.py

```diff
@@ -0,0 +1,45 @@
+# coding=utf-8
+
+# Copyright (c) 2001, Canal TP and/or its affiliates. All rights reserved.
+#
+# This file is part of Navitia,
+#     the software to build cool stuff with public transport.
+#
+# Hope you'll enjoy and contribute to this project,
+#     powered by Canal TP (www.canaltp.fr).
+# Help us simplify mobility and open public transport:
+#     a non ending quest to the responsive locomotion way of traveling!
+#
+# LICENCE: This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# [matrix] channel #navitia:matrix.org (https://app.element.io/#/room/#navitia:matrix.org)
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+
+from flask_restful import Resource
+
+from kirin.core.types import ConnectorType
+from kirin.queue_workers.http_utils import post_feed, get_contributors_by_connector_type
+
+
+class PivIndex(Resource):
+    def get(self):
+        return get_contributors_by_connector_type(ConnectorType.piv.value)
+
+
+class Piv(Resource):
+    def post(self, contributor_id=None):
```

Comment:
Do we really need a default value for `contributor_id`? What does it mean to call `post` without a `contributor_id`? Actually, if I understand well, this will lead to an `abort(400)`. Maybe we should remove the default value.
```suggestion
    def post(self, contributor_id):
```

That said, we also have the following comment in `kirin/queue_workers/piv/piv_worker.py`.

```python
    # We assume a maximum of one PIV contributor is going to exist in the DB
    # Therefore, we enforce a maximum of one `PivWorker` is running at a time
```

Reply:
If flask output's a message on missing contributor that's clear, I'm OK to remove the default value.

Removing the whole param doesn't seem like a good path, as the idea is more to provide handling of multiple contributors for the same connector than the opposite (the need just doesn't exist currently on those like it exists for GTFSRT... and it's a important architectural work to do).
It's also more consistent with GTFSRT.
Finally, it's about to be refactored in a separate PR to be handled once and for all (if possible).